src/prompts/index.tsx

```diff
@@ -46,8 +43,6 @@ export class PromptManager {
     private pushOpsManager?: PushOpsManager;
     private ui?: Ui;
     private prompts: PromptConfigs<PushPromptConfig>;
-    private currentPostAction?: PromptAction;
```

Comment:
can remove remaining references to 'postaction' ?

Reply:
Removed